ibis/expr/api.py

```diff
@@ -4126,7 +4126,7 @@ def f(self, other, predicates=()):
     return f
 
 
-def filter(table, predicates):
+def _filter(table, predicates):
```

Comment:
i don't think this is needed here

Reply:
I can make this a separate PR if you want.  It's not "needed" but it's not a good idea in general to override a Python builtin--even confined to file scope, it prevent the builtin from being usable.

Reply:
its not about a separate PR, what does this fix? sure its a builtin, but so what

Reply:
It doesn't fix anything, but makes the use of filter inside the module less surprising, which is reasonable. I think this change is fine.